internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -659,3 +663,39 @@ func convertTopicSubscription(t manifest.TopicSubscription, validTopicARNs []str
 		Service: aws.String(t.Service),
 	}, nil
 }
+
+func convertRDWkldCustomResources(customDomainURLs map[string]string) (bucket *string, urls map[string]*string, err error) {
```

Comment:
I wasn't super sure about this either. I feel like in general an optional fields should have a pointer type for that reason (`{{if .ScriptBucketName}}`)

However, those fields - `ScriptBucketName`, DNSName`, etc. - are required if `Alias` is non-zero. So codes like `{{if .ScriptBucketName}}` shouldn't need to appear in the template, only `{{if .Alias}}`.

But since `Alias` is optional, that automatically makes all of the new fields optional too.. hence pointer.